src/renderer.zig

```diff
@@ -112,15 +112,21 @@ var worldFrameBuffer: graphics.FrameBuffer = undefined;
 var lastWidth: u31 = 0;
 var lastHeight: u31 = 0;
 var lastFov: f32 = 0;
+var lastFovExtra: f32 = 0;
 pub fn updateViewport(width: u31, height: u31, fov: f32) void {
```

Comment:
This function must not touch the fov values inserted. It is for example used by the background scene recording, which requires a fov of 90°.